features/device_key/source/DeviceKey.cpp

```diff
@@ -259,22 +246,22 @@ int DeviceKey::get_derived_key(uint32_t *ikey_buff, size_t ikey_size, const unsi
     return DEVICEKEY_SUCCESS;
 }
 
-int DeviceKey::generate_key_by_random(uint32_t *output, size_t size)
+int DeviceKey::device_generate_root_of_trust()
```

Comment:
I know I proposed this name, but I'm starting to wonder.. is there and extra "device" in the `DeviceKey::device_do_something...`

Maybe the `DeviceKey::generate_root_of_trust()` should be enough.

Reply:
Renamed